pkg/expr/sql/db_test.go

```diff
@@ -193,6 +193,18 @@ func TestQueryFramesDateTimeSelect(t *testing.T) {
 	}
 }
 
+func TestErrorsFromGoMySQLServerAreFlagged(t *testing.T) {
+	const GmsNotImplemented = "STDDEV" // not implemented in go-mysql-server as of 2025-03-18
+
+	db := DB{}
+
+	query := `SELECT ` + GmsNotImplemented + `(1);`
+
+	_, err := db.QueryFrames(context.Background(), "sqlExpressionRefId", query, nil)
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "error in go-mysql-server")
```

Comment:
Do you want me to test for the other case too - ie. when the error isn't `FunctionNotFound` then we don't see this string in the message?